bughouse_console/src/stats_handlers_tide.rs

```diff
@@ -441,3 +458,23 @@ fn format_timestamp_date_and_time(maybe_ts: Option<OffsetDateTime>) -> Option<(S
         .ok()?;
     Some((date, time))
 }
+
+fn keep_last_point_per_date(points: &mut Vec<RawStats>) {
+    if points.is_empty() {
+        return;
+    }
+    let n = points.len();
+    let mut j = 0;
+    let mut prev_date = None;
+    for i in 1..n {
+        let date = points[i].last_update.map(OffsetDateTime::date);
+        if prev_date.is_some() && date != prev_date {
```

Comment:
IIUC you discard `None` date in the beginning, but keep `None` after `Some`. What is the logic behind this? How about discarding missing dates altogether?

Reply:
Sounds good, done (by using your code from comment above).

There wasn't really much logic behind it (i.e. I I didn't really care what happens to the updates w/o update time). I think they wouldn't be displayed well anyway, so filtering looks correct.